src/Component/RegistrationForm.js

```diff
@@ -0,0 +1,278 @@
+import React, { Component } from 'react'
+import '../Stylesheet/RegistrationForm.css'
+
+class RegistrationForm extends Component {
+  constructor(props) {
+    super(props)
+    this.isInputsValid = true
+    this.state = {
+      loginId: {
+        value:'',
+        error:''
+      },
+      email: {
+        value:'',
+        error:''
+      },
+      name: {
+        value:'',
+        error:''
+      },
+      url: {
+        value:'',
+        error:''
+      },
+      timezone: {
+        value:'SelectTimezone',
```

Comment:
this should not be a value. it should be a prompt in select box

Reply:
Done